### PR TITLE
[core] use std:: instead of using namespace std

### PR DESCRIPTION
**Pull request by mkortstiege**
Monkey`d through them after running search/replace, but I bet there are some compile errors ;)

### CODE REVIEW

**Comment by Paxxi** [comment]
I'm not even remotely bored enough to review, I say if it builds on all platforms then it's good to merge :)

**Comment by mkortstiege** [comment]
> I'm not even remotely bored enough to review, I say if it builds on all platforms then it's good to merge :)

Haha ;) Will give it a go once jenkins is actually able to handle building again.

File: xbmc/addons/Scraper.cpp [hunk 1]
```diff
@@ -238,10 +237,10 @@ void CScraper::ClearCache()
 // is XML output by chained functions, possibly recursively
 // the CCurlFile object is passed in so that URL fetches can be canceled from other threads
 // throws CScraperError abort on internal failures (e.g., parse errors)
-vector<string> CScraper::Run(const std::string& function,
+std::vector<std::string> CScraper::Run(const std::string& function,
                                  const CScraperUrl& scrURL,
```
Comment (tamland) [inline xbmc/addons/Scraper.cpp]: why i hate this indent style. it always screws up formatting when refactoring:\

Comment (mkortstiege) [inline xbmc/addons/Scraper.cpp]: Will re-indent.

**Comment by mkortstiege** [comment]
jenkins build this please.

**Comment by mkortstiege** [comment]
Rebased. jenkins build this please.

**Comment by mkortstiege** [comment]
jenkins build this please (again).

**Pull request merged**